src/docs/sdk/performance/span-operations.mdx

```diff
@@ -27,8 +27,8 @@ If a span operation is not provided, the value of `default` is used.
 |            | resource.img    |                                                                                                                     |
 | browser    |                 | Usage of browser APIs or functionality                                                                              |
 |            | browser.paint   |                                                                                                                     |
-|            | browser.mark    | Usage of [performance.mark() API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark)                |
-|            | browser.measure | Usage of [performance.measure() API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure)          |
+| mark       |                 | Usage of [performance.mark() API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark)                |
+| measure    |                 | Usage of [performance.measure() API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure)          |
```

Comment:
Hmm I feel like I could live without this commit; but you adding it to this PR suggests the distinction between `Category` and `Usage` columns is not really necessary.

All we have as actual operations in SDKs are strings, and since we've dropped the intention to document how categories work we could as well have a single column:

| Operation | Description |
| --- | --- |
| ... | ... |

Reply:
Will address this in the next PR.